scripts/github_bot/README.md

```diff
@@ -0,0 +1,78 @@
+GitHub bot for Azure CLI
+========================
+
+Description
+-----------
+This is a GitHub bot/webhook for automated component/package releases.
+
+The server can be extended to handle more GitHub webhook events in the future as needed.
+
+
+Run locally with `export FLASK_APP=app.py; flask run -h '0.0.0.0'`
+
+How to Build
+------------
+```
+$ sudo docker build --no-cache -t azure-cli-bot-api:1 .
+```
+
+How to Run with Docker
+----------------------
+The following environment variables are required for the server to run:
+
+`GITHUB_SECRET_TOKEN` - The Secret that is configured in GitHub Webhook settings.  
+`ALLOWED_USERS` - Space separated list of GitHub usernames that can create releases.  
+`PYPI_REPO` - URL to PyPI (e.g. https://testpypi.python.org/pypi or https://pypi.python.org/pypi).  
+`TWINE_USERNAME` - Username to authenticate with PyPI.  
+`TWINE_PASSWORD` - Password to authenticate with PyPI.
+
+For example:
+
+```
+$ sudo docker run -d -e "GITHUB_SECRET_TOKEN=<secret>" -e "ALLOWED_USERS=blue red yellow" \
+-e "PYPI_REPO=https://testpypi.python.org/pypi" -e "TWINE_USERNAME=<user>" -e "TWINE_PASSWORD=<pass>" \
```

Comment:
Yeah it will but this code will be running on a server behind security to prevent access.
The server will have to know the password at some point in order to be able to publish the package.